bin/interop-decode.c

```diff
@@ -350,10 +350,11 @@ main (int argc, char **argv)
         }
         else
         {
+        dec_header:
             p = buf->buf + buf->off;
             if (buf->off == 0)
                 rhs = lsqpack_dec_header_in(&decoder, buf, buf->stream_id,
-                                buf->size, &p, buf->size, &hset, NULL, NULL);
+                                buf->size, &p, buf->size / 2, &hset, NULL, NULL);
```

Comment:
This is interesting: what happens if `buf->size` is 1?

Reply:
Interesting case indeed.

I have tested this now and a `buf->size == 1` fails. However, it fails on commits that predate this one. Meaning, `upstream/master` will fail if this `1` is added to _test/qifs/fb-req.qif_ like so:
```
1

:path	/rsrc.php/v3/yn/r/rIPZ9Qkrdd9.png
accept-encoding	gzip, deflate, br
accept-language	en-US,en;q=0.9
...
```